### DISCUSSION TITLE
目录怎么才能带数字？

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#115] [Q&A]
你好，生成的目录只有标题，没有标题前面的数字，怎么修改一下，才能带上数字呢？

**Comment by tanukihee** [comment]
代码 & 图

建议你按 issue 模板提问

**Reply by earthmoon** [reply]
> 代码 & 图
> 
> 建议你按 issue 模板提问

对不起，是我搞错了，目前前面有数字，是pdf中导航中没有数字。我以为都没有数字。
![Screenshot 2022-04-12 203459](https://user-images.githubusercontent.com/45897471/162963512-56ed8f8e-8364-4322-a25f-bf9ff827f4ca.jpg)

**Reply by sikouhjw** [reply]
加到导言区 `\hypersetup{bookmarksnumbered=true}`

**Reply by earthmoon** [reply]
> \hypersetup{bookmarksnumbered=true}

搞定，谢谢！
